src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -49,6 +50,11 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         try {
             $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
         } catch (\Exception $e) {
+        } catch (\Throwable $e) {
+            $e = new FatalThrowableError($e);
```

Comment:
> is this wrapper really necessary ?

Would it be better to let it bubble up?

> the class may need to be altered

You mean the if's that prefix the message?